it/google-cloud-platform/src/main/java/com/google/cloud/teleport/it/gcp/LoadTestBase.java

```diff
@@ -268,22 +267,17 @@ private void computeDirectMetrics(
   protected Map<String, Double> getMetrics(
       LaunchInfo launchInfo, @Nullable String inputPcollection, @Nullable String outputPcollection)
       throws InterruptedException, IOException, ParseException {
-    if ("DataflowRunner".equalsIgnoreCase(launchInfo.runner())) {
-      // Dataflow runner specific metrics
-      // In Dataflow runner, metrics take up to 3 minutes to show up
+    Map<String, Double> metrics = pipelineLauncher.getMetrics(project, region, launchInfo.jobId());
```

Comment:
Is these metrics immedately available now (without waiting for 4 minutes)?

Reply:
The dataflow metrics from getMetrics API is available immediately. However, the monitoring metrics take upto 4 minutes to appear.

I just moved the `Thread.sleep()` before querying monitoring metrics.

Reply:
sounds good, then possibly comment "// In Dataflow runner, monitoring metrics take up to 3 minutes to show up"